.github/pull-request-template.md

```diff
@@ -0,0 +1,12 @@
+### Describe your PR
+
+
+<!-- Please refer to the [CONTRIBUTING.md](./CONTRIBUTING.md) for detailed instructions on how to fill out this section. -->
```

Comment:
I think we should give more instructions and examples here. Most of the people won't bother to copy the link.

And top of that you can embed the section as an anchor. Mylink#my-section-with-hyphens

Reply:
Added comments and examples ✅

In my recent [commit](https://github.com/prometheus/client_golang/pull/1518/commits/37203780203ac14c0879ef5aef3da61b3b2130d9), I updated the auto-PR-label configuration file. Previously, the regex was incorrectly capturing labels from commented examples, which has now been corrected.